CHANGELOG.md

```diff
@@ -18,10 +47,10 @@ All notable changes to this project will be documented in this file.
 * Chart hooks for managing LTPA keys and MQ client user
 * Liberty runtime liveness probe (checks log for specific error messages)
 * Instructions for handling failed JMS messages on the MQ dead message queue
-* In the MQ chart, a check before creating deployment to see if multi-instance MQ is desired.
-* Values for use in multi-instance MQ, with both static and dynamic storage.
-* A stateful set YAML file for use in multi-instance MQ.
-* PV and PVC YAML files for use in multi-instance MQ.
```

Comment:
Is there a reason for the removal of the full stops?

Reply:
They were removed for consistency with the other listed changes